packages/fast-usdc/src/util/bank.js

```diff
@@ -0,0 +1,14 @@
+/* global globalThis */
```

Comment:
in general prefer `/* eslint-env node */` to indicate the environment a module expects.

see https://github.com/Agoric/agoric-sdk/pull/8702

Reply:
Are they mutually exclusive? I want to use `globalThis` to namespace the global fetch, and have the arg default to that. I guess I'd have to rename it to `fetchArg` or something. But either way, I removed the default arg as you said, so it's moot.

Reply:
>Are they mutually exclusive? 

You mean is it okay to have both these lines?
```
/* global globalThis */
/* eslint-env node */
```

It doesn't cause harm but the first is accomplished by the second so it gives the reader pause as to why it's included.